api/candlepin-api-spec.yaml

```diff
@@ -2432,55 +2735,1142 @@ paths:
+      security: []
+      responses:
+        200:
+          description: Owner info successfully retrieved
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OwnerInfoDTO'
+        404:
+          description: An owner could not be found using the provided key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: An owner could not be found using the provided key
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /owners/{owner_key}/log:
+    put:
+      description: Sets the log level for an owner
+      tags:
+        - Owner log
+      operationId: setLogLevel
+      security: []
+      parameters:
+        - name: owner_key
+          in: path
+          description: The key of the owner
+          required: true
+          schema:
+            type: string
+        - name: level
+          in: query
+          description: The log level
+          required: true
+          schema:
+            type: string
+            default: DEBUG
+            enum:
+              - ALL
+              - TRACE
+              - DEBUG
+              - INFO
+              - WARN
+              - ERROR
+              - OFF
+      responses:
+        200:
+          description: Log level successfully set
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OwnerDTO'
+        400:
+          description: |
+            Provided level is not a valid log level
+            or owner key is not provided
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: |
+                  Provided level is not a valid log level
+                  or owner key is not provided
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        404:
+          description: An owner could not be found using the provided key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: An owner could not be found using the provided key
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    delete:
+      description: Remove the log level of an owner
+      tags:
+        - Owner log
+      operationId: deleteLogLevel
+      parameters:
+        - name: owner_key
+          in: path
+          description: The owner key
+          required: true
+          schema:
+            type: string
+      security: []
+      responses:
+        200:
```

Comment:
Please change the response code to ```204``` instead of 200.

Reply:
Changed the response code to 204.